src/PrestaShopBundle/Form/Admin/Type/NumberMinMaxFilterType.php

```diff
@@ -47,6 +49,22 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults([
             'min_field_options' => [],
             'max_field_options' => [],
+            'constraints' => [
+                new Callback([
+                    'callback' => function (?array $impactData, ExecutionContextInterface $context) {
+                        if (!isset($impactData['min_field']) || !isset($impactData['max_field'])) {
+                            return;
+                        }
+
+                        if ($impactData['min_field'] > $impactData['max_field']) {
```

Comment:
isn't there a risk for inaccurate float comparisons? Maybe we should use DecimalNumber here to compare? (also maybe then we could reuse the same constraint in both integer and number MinMaxFilterTypes?

Reply:
Yeah I guess it would be wprth a refacto to integrate those improvements, if there are some bugs found I'll do it in this PR but I prefer not to book it for now